packages/react/src/components/ui2/buttons/SecondaryButton.tsx

```diff
@@ -1,18 +1,40 @@
 import type { FC, PropsWithChildren } from 'react';
 import React from 'react';
 
+import { Text } from '../typography/Text';
+
 export interface SecondaryButtonProps {
+  variant?: 'primary' | 'secondary';
   className?: string;
+  disabled?: boolean;
   onClick: () => void;
 }
 
-export const SecondaryButton: FC<PropsWithChildren<SecondaryButtonProps>> = ({ className, children, onClick }) => {
+export const SecondaryButton: FC<PropsWithChildren<SecondaryButtonProps>> = ({
+  variant = 'primary',
```

Comment:
the primary vs. secondary variant confused my a bit (we also have a PrimaryButton and a SecondaryButton). What does primary vs. secondary mean here?

Reply:
so this means primary vs secondary color.. as in to use which css variable.. and the PrimaryButton and SecondaryButton refer to the styling. I will the field from variant to colorVariant and make the values `primary` | `link`